tests/factories.py

```diff
@@ -718,3 +719,69 @@ def _create(cls, *args, **kwargs):
             )
         )
         return super(NodeLicenseRecordFactory, cls)._create(*args, **kwargs)
+
+
+def render_generations_from_parent(parent, creator, num_generations):
+    current_gen = parent
+    for generation in xrange(0, num_generations):
+        next_gen = NodeFactory(
+            parent=current_gen,
+            creator=creator,
+            title=fake.science_sentence(),
```

Comment:
`fake` here is just `faker.Factory`, which doesn't normally have `.science_<things>`. This looks like it came from `create_fakes.py`, which also called `fake.add_provider(Sciencer)`.